src/videojs-hls.js

```diff
@@ -558,6 +555,11 @@ videojs.Hls.prototype.fillBuffer = function(offset) {
     segment,
     segmentUri;
 
+  // if preload is set to "none", do not download segments until playback is requested
+  if (!player.hasClass('vjs-has-started') && player.options.preload === "none") {
```

Comment:
Should the second condition be `player.preload() === 'none'`? `options` are set on player creation and (I believe) ignored from that point out.